### PR TITLE
feat: add dependency id to event hubs dep tracking

### PR DESCRIPTION
**Pull request by stijnmoreels**
Adds dependency ID to Azure EventHubs dependency tracking.

Relates to #243

### CODE REVIEW

**Review by fgheysels — APPROVED** [review]

File: src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerHttpDependencyExtensions.cs [hunk 1]
```diff
@@ -19,10 +19,10 @@ public static partial class ILoggerExtensions
         /// Logs an HTTP dependency
         /// </summary>
         /// <param name="logger">The logger to track the telemetry.</param>
-        /// <param name="request">Request that started the HTTP communication</param>
-        /// <param name="statusCode">Status code that was returned by the service for this HTTP communication</param>
-        /// <param name="measurement">Measuring the latency of the HTTP dependency</param>
-        /// <param name="context">Context that provides more insights on the dependency that was measured</param>
+        /// <param name="request">The request that started the HTTP communication.</param>
+        /// <param name="statusCode">The status code that was returned by the service for this HTTP communication.</param>
+        /// <param name="measurement">The measuring the latency of the HTTP dependency.</param>
```
[the remark this answers is not in the document]
Comment (stijnmoreels) [reply src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerHttpDependencyExtensions.cs:24]: Don't think this is much a change to readability? It is still the latency that it's supposed to test, the type we use is called the duration measurement. Not sure if this is an improvement.

[the remark this answers is not in the document]
Comment (stijnmoreels) [reply src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerHttpDependencyExtensions.cs:24]: Also, this is the same description on all dependency tracking.

[the remark this answers is not in the document]
Comment (fgheysels) [reply src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerHttpDependencyExtensions.cs:24]: I just think the description 'The measuring the' is kind off strange :)
Then it should be 'The measuring of the latency' or 'The measurement of the latency'
```suggestion
        /// <param name="measurement">The measurement of the latency of the HTTP dependency.</param>
```

[the remark this answers is not in the document]
Comment (stijnmoreels) [reply src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerHttpDependencyExtensions.cs:24]: Yes, let's go with that! 👍

**Pull request merged**